File: 0392-is-subsequence/0392-is-subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:

        #Pointers
        pointer_s = 0
        pointer_t = 0

        len_s = len(s)
        len_t = len(t)

            #Iterate through prime sequence
        while pointer_s < len_s and pointer_t < len_t:
                
                if s[pointer_s] == t[pointer_t]:
                #value of s and only pointer_t +1
                    pointer_s += 1
                #else both pointer +1
                pointer_t += 1
        #return boolean value 
        return pointer_s == len_s

File: 0392-is-subsequence/0392-is-subsequence.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        //Define pointers and length of strings
        int pointerS = 0;
        int pointerT = 0;

        int lengthS = s.length();
        int lengthT = t.length();

        while(pointerS < lengthS && pointerT < lengthT){
            if(s.charAt(pointerS) == t.charAt(pointerT)){
                pointerS++;
            }
            pointerT++;
        }

        return pointerS == lengthS;
    }
}